### ISSUE TITLE
Clearing cache

### ISSUE THREAD

**Issue by MaxArt2501** [danielhusar/gulp-save#5]
Hi,
I was close to make myself a plugin like this, but then I found yours and noticed this is *exactly* what I had in mind. Nice!

One thing I'd suggest is the ability to clear the cache, because for large numbers of files and longer tasks it may nice to have memory freed as the task proceed. This comes especially useful whe manipulating images or other large files.

I had two strategies in mind: one would be adding a second, boolean argument to `restore`, like this:

```js
module.exports.restore = function (store, clear) {
    ...
    cache[store].forEach(function (file) {
        this.push(file);
    }.bind(this));

    if (clear) {
        delete cache[store];
    }
    ...
};
```

The other approach would be adding a `clear` method to the plugin, that clears the *whole* cache (or maybe just a selection of stores):

```js
module.exports.clear = function () {
    var stores;
    if (arguments.length) {
        stores = Array.prototype.slice.call(arguments);
    } else {
        stores = Object.keys(cache);
    }

    stores.forEach(function(store) {
        delete cache[store];
    };
};
```

What do you think about it?

**Comment by danielhusar** [comment]
Hey,

Im happy to accept PRs, your second approach looks good to me.
If you are too busy I can implement it during weekend.

Thanks.

**Comment by MaxArt2501** [comment]
Closed with #7.
